src/components/CameraButtonSvg.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import colors from '../assets/colors.js';
+import { Svg } from 'expo';
+
+class CameraButtonSVG extends React.Component {
+  render() {
+    const { Circle } = Svg;
+
+    return (
+      <Svg height="100" width="100">
+        <Circle
+          cx="50"
+          cy="50"
+          r="30"
+          stroke={colors.white}
+          strokeWidth="5"
+          fill="none"
+        />
+
+        <Circle
+          cx="50"
+          cy="50"
+          r="25"
+          strokeWidth="0"
+          fill={colors.white}
+        />
+      </Svg>
```

Comment:
whoa, weird that we have to build this ourselves. Is there not an easy way to get the native controls for the camera?